qa/workunits/rbd/rbd_mirror_helpers.sh

```diff
@@ -795,6 +795,32 @@ get_newest_mirror_snapshot()
         ${log} || true
 }
 
+get_newest_complete_mirror_snapshot_id_on_primary()
+{
+    local primary_cluster=$1
```

Comment:
I don't see anything specific to whether the cluster is considered primary in what this helper does:
```suggestion
get_newest_complete_mirror_snapshot_id()
{
    local cluster=$1
```